doc/testing.md

```diff
@@ -638,6 +640,29 @@ double click on it; enable or disable hotkeys.
 
 Note: restart is required to make the settings take effect.
 
+#### Robot API
+
+Most part of automated Client UI tests use Robot API to control the UI. Usually
+the default operating system settings need to be adjusted for correct work of
+Robot functionality. The detailed steps how to access and update that settings
+for different platforms are provided below.
+
+##### MacOS
+
+Robot functionality is not permitted to control your Mac by default starting
+from MacOS 10.15. To enable it choose Apple menu; System Settings, click
+Privacy & Security; select Accessibility and ensure the following apps are
+allowed to control your computer: `Java` and `Terminal`. If the tests are run
+from an IDE, the IDE should be granted this permission too.
```

Comment:
```suggestion
`Robot` is not permitted to control your Mac by default since
macOS 10.15. To enable it, choose Apple menu > System Settings, click
Privacy & Security, then click Accessibility and ensure the following apps are
allowed to control your computer: *Java* and *Terminal*. If the tests are run
from an IDE, the IDE should be granted this permission too.
```
I amended the instructions align with [Allow accessibility apps to access your Mac](https://support.apple.com/en-gb/guide/mac-help/mh43185/mac).

Perhaps, “to allow it” is better than “to enable it”. Also unsure about “`Robot` is not *allowed* to control…” instead of “permitted”.

*Java* and *Terminal* are the names of apps, not part of a command-line, therefore they're shouldn't be rendered in monospace font; using italics suits better to separate the names from regular text (even though it's not required).

Reply:
Fixed